techniques/aria/ARIA17.html

```diff
@@ -1,84 +1,110 @@
-    &lt;p id="alert1"&gt;Send an alert when balance exceeds $ 3,000&lt;/p&gt;
-    &lt;div&gt;
-      &lt;span role="radio" aria-labelledby="a1r1" name="a1radio"&gt;&lt;/span&gt;
-      &lt;span id="a1r1"&gt;Yes&lt;/span&gt;
-    &lt;/div&gt;
-    &lt;div&gt;
-      &lt;span role="radio" aria-labelledby="a1r2" name="a1radio"&gt;&lt;/span&gt;
-      &lt;span id="a1r2"&gt;No&lt;/span&gt;
-    &lt;/div&gt;
+      
+         <p>This example demonstrates use <code class="prop">role=radiogroup</code>.</p>
+         <div class="note">
+            <p>The radio buttons are custom controls that use <code class="prop">role=radio</code>, but the script to make them work like HTML radio buttons is not included in this example.</p>
+         </div>
+<pre xml:space="preserve">&lt;h3&gt;Set Alerts for your Account&lt;/h3&gt;
+&lt;div role="radiogroup" aria-labelledby="alert1"&gt;
+  &lt;p id="alert1"&gt;Send an alert when balance exceeds $3,000&lt;/p&gt;
+  &lt;div&gt;
+    &lt;span role="radio" aria-checked="false" aria-labelledby="a1r1"&gt;&lt;/span&gt;
+    &lt;span id="a1r1"&gt;Yes&lt;/span&gt;
+  &lt;/div&gt;
+  &lt;div&gt;
+    &lt;span role="radio" aria-checked="false" aria-labelledby="a1r2"&gt;&lt;/span&gt;
+    &lt;span id="a1r2"&gt;No&lt;/span&gt;
   &lt;/div&gt;
-  &lt;div role="radiogroup" aria-labelledby="alert2"&gt;
-    &lt;p id="alert2"&gt;Send an alert when a charge exceeds $ 250&lt;/p&gt;
-    &lt;div&gt;
-      &lt;span role="radio" aria-labelledby="a2r1" name="a2radio"&gt;&lt;/span&gt;
-      &lt;span id="a2r1"&gt;Yes&lt;/span&gt;
-    &lt;/div&gt;
-    &lt;div&gt;
-      &lt;span role="radio" aria-labelledby="a2r2" name="a2radio"&gt;&lt;/span&gt;
-      &lt;span id="a2r2"&gt;No&lt;/span&gt;
-    &lt;/div&gt;
+&lt;/div&gt;
+&lt;div role="radiogroup" aria-labelledby="alert2"&gt;
+  &lt;p id="alert2"&gt;Send an alert when a charge exceeds $250&lt;/p&gt;
+  &lt;div&gt;
+    &lt;span role="radio" aria-checked="false" aria-labelledby="a2r1"&gt;&lt;/span&gt;
+    &lt;span id="a2r1"&gt;Yes&lt;/span&gt;
   &lt;/div&gt;
-  &lt;p&gt;&lt;input type="submit" value="Continue" id="continue_btn" name="continue_btn" /&gt;&lt;/p&gt;</pre>
-            <p>Related CSS Style Definition to place a border around the group of fields :</p>
-            <pre xml:space="preserve">div[role=radiogroup] {
-  border: black thin solid;
-} </pre>
-            <p class="working-example">Working example: <a href="../../working-examples/aria-grouping-related-fields/">using grouping roles to identify related form controls</a>.</p>
+  &lt;div&gt;
+    &lt;span role="radio" aria-checked="false" aria-labelledby="a2r2"&gt;&lt;/span&gt;
+    &lt;span id="a2r2"&gt;No&lt;/span&gt;
+  &lt;/div&gt;
+&lt;/div&gt;
+&lt;input type="submit" value="Continue" id="continue_btn" name="continue_btn" /&gt;</pre>
+
+   <p class="working-example">Working example: <a href="../../working-examples/aria-grouping-related-fields/">using grouping roles to identify related form controls</a>.</p>
```

Comment:
I don't know how WCAG working example folder works. We can reuse simplified [ARIA APG Radio group](https://www.w3.org/WAI/ARIA/apg/example-index/radio/radio-activedescendant.html) example if needed.

Reply:
have updated the document to reference this.